core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -1357,7 +1357,7 @@ private[spark] object JsonProtocol {
           message.getOrElse("Unknown reason"))
       case `exceptionFailure` =>
         val className = json.get("Class Name").extractString
-        val description = json.get("Description").extractString
+        val description = jsonOption(json.get("Description")).map(_.extractString).orNull
         val stackTrace = stackTraceFromJson(json.get("Stack Trace"))
```

Comment:
Are we keeping this change ? In the discussion above, you mentioned you are reverting this ?